### ISSUE TITLE
Uncaught TypeError: Cannot set property 'query' of undefined

### ISSUE THREAD

**Issue by jensellfors** [birkir/gatsby-source-prismic-graphql#62]
I'm getting this JS error when running in production ie `gatsby build && gatsby serve`. (`gatsby develop` works fine.)

This issue seems to be fixed in this commit: https://github.com/birkir/gatsby-source-prismic-graphql/commit/f05a5e62ec6c9dec483e127192073de5cc85907c

But since it hasn't been released and there is no issue I thought I'd add one to help other people with the same errors.

```
app-81af51e12fde343aa4e9.js:1 Uncaught TypeError: Cannot set property 'query' of undefined
    at Object.<anonymous> (app-81af51e12fde343aa4e9.js:1)
    at u (webpack-runtime-17c45135f10339e5f97e.js:1)
    at Object.<anonymous> (app-81af51e12fde343aa4e9.js:1)
    at u (webpack-runtime-17c45135f10339e5f97e.js:1)
    at Object.<anonymous> (app-81af51e12fde343aa4e9.js:1)
    at u (webpack-runtime-17c45135f10339e5f97e.js:1)
    at Module.<anonymous> (app-81af51e12fde343aa4e9.js:1)
    at u (webpack-runtime-17c45135f10339e5f97e.js:1)
    at r (webpack-runtime-17c45135f10339e5f97e.js:1)
    at Array.t [as push] (webpack-runtime-17c45135f10339e5f97e.js:1)
```

**Comment by traaidmark** [comment]
I am running into this issue big time. Is there perhaps a possibility this can be released, or alternatively if you guys can tell me how I can get it patched or added to my site?

Sorry git is a bit of a dark hole for me, and I don't want to be unhelpful

**Comment by jensellfors** [comment]
I can confirm that this issue got fixed in `v3.4.0-beta.1`. This works for me:

```
  "gatsby": "^2.13.37",
  "gatsby-source-prismic-graphql": "3.4.0-beta.1",
   ...
```
